docs/book/v2/migration/preparing-for-v3.md

```diff
@@ -5,6 +5,27 @@ This document is intended to help you prepare for these changes.
 
 ## Removed Features
 
+### `laminas/laminas-cache` Removal
+
+From version 3.0 onwards, `laminas-session` will remove the `laminas/laminas-cache`
+dependency and replace it with `psr/simple-cache` to adhere to PSR-16.
+
+The `Laminas\Session\SaveHandler\Cache.php` class will be updated to reflect this change.
+Starting from version 3.0 the class will be made `final` and has been marked as such in the current version.
```

Comment:
This does not belong under the headline "Removed Features". If you want to list the fact under preparation for the next major version, then generally point out that all classes which already marked as final will become final. However, this must be done under a separate headline like "Upcoming Behaviour & Signature Changes".